### ISSUE TITLE
Web support

### ISSUE THREAD

**Issue by loic-hamdi** [jorgefspereira/persona_flutter#28]
Are you planning to integrate web support?

**Comment by jorgefspereira** [comment]
@loic-hamdi Yes. In the upcoming months I'll probably need to integrate persona web on a professional project. I'll take that opportunity to update this plugin with web support.

**Comment by loic-hamdi** [comment]
Sounds great! 
I was wondering if Persona was actually supporting you on the package development :) 
Thanks for the great work 🙌

**Comment by loic-hamdi** [comment]
Hi @jorgefspereira , are you still planning of releasing a web compatible version?
Thank you

**Comment by jorgefspereira** [comment]
@loic-hamdi unfortunately not at moment. For now I only have time to keep the library updated. Feel free to contribute with that feature.
